app/scripts/modules/core/src/application/applicationSearchResultType.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from 'react';
+
+import {
+  AccountCell, BasicCell, HrefCell, searchResultTypeRegistry, ISearchResult,
+  HeaderCell, TableBody, TableHeader, TableRow, SearchResultTab, ISearchColumn,
+} from 'core/search';
+
+export interface IApplicationSearchResult extends ISearchResult {
+  accounts: string[];
+  application: string;
+  cloudProviders: string;
+  createTs: string;
+  description: string;
+  email: string;
+  group: string;
+  lastModifiedBy: string;
+  legacyUdf: boolean;
+  name: string;
+  owner: string;
+  pdApiKey: string;
+  updateTs: string;
+  url: string;
+  user: string;
+}
```

Comment:
Are all these fields necessary or relevant to the application search result? It reads as a list of attributes stored in Front50 for an application, wondering if it's better to declare this interface as `IApplicationAttributes` and have the application model object implement it or something. Or not, who really knows.